src/lib/jetlore.js

```diff
@@ -0,0 +1,18 @@
+/* @flow */
```

Comment:
could you add additional comment to state where this asset is pulled from? (Jetlore's asset url)

Reply:
Would it make more sense to load this resource from paypalobjects rather than inlining it?

Reply:
This is meant to be integrated with all the tracking events so Arvind Naidu prefers to bundle this in, unless you insist otherwise. Do you insist @bluepnume ?